unittests/CppInterOp/InterpreterTest.cpp

```diff
@@ -122,6 +125,9 @@ TEST(InterpreterTest, ActivateInterpreter) {
 }
 
 TEST(InterpreterTest, Process) {
+#ifdef EMSCRIPTEN_STATIC_LIBRARY
+  GTEST_SKIP() << "Test fails for Emscipten static library build";
+#endif
```

Comment:
Like literally Process which is at heart of xeus-cpp's code execution fails ? 

Not sure if it makes sense to skip this out of all the possible tests :)